packages/orchestration/src/examples/unbond.contract.js

```diff
@@ -27,8 +27,14 @@ import * as flows from './unbond.flows.js';
  * @param {Zone} zone
  * @param {OrchestrationTools} tools
  */
-const contract = async (zcf, privateArgs, zone, { orchestrateAll }) => {
-  const { unbondAndLiquidStake } = orchestrateAll(flows, { zcf });
+const contract = async (
+  zcf,
+  privateArgs,
+  zone,
+  { orchestrateAll, zcfForFlows },
+) => {
+  // TODO: figure out Guarded<ZcfForFlows> vs. ZcfForFlows
+  const { unbondAndLiquidStake } = orchestrateAll(flows, { zcfForFlows });
```

Comment:
@turadg I'm still interested in help with the types here.

Reply:
I took a look and the best fix isn't obvious to me. Some of the problem may be going from guest to host and back. 

I would try defining the interface as a Guest interface (akin to ZCF) and making that what the flow takes. Then define the Exo in terms of that using `HostOf`.

Reply:
ok, I did something like that. Now it seems to be griping about brands etc. that come from `getTerms()` -- their `isMyIssuer` method returns a promise, and it wants vows for those. Clues?